### ISSUE TITLE
[BUG]: wierd slider - it isnt on the same level - its on different level

### ISSUE THREAD

**Issue by inson1** [martinrotter/rssguard#1187]
### Brief description of the issue

Bug
![image](https://github.com/martinrotter/rssguard/assets/75314629/8e1cf84f-d354-47ac-9f93-46dc31f18fb9)

Icons
![image](https://github.com/martinrotter/rssguard/assets/75314629/a04db9fe-4a01-49bc-8e32-a86577894f3e)

Interesting 
![image](https://github.com/martinrotter/rssguard/assets/75314629/45b0892e-aa53-40a9-a436-3d904b25ea1f)

### How to reproduce the bug?

.

### What was the expected result?

.

### What actually happened?

.

### Debug log

.

### Operating system and version

Windows 10
4.5.3

**Comment by martinrotter** [comment]
For christ sake: Sorry man, I will try to be polite. STOP posting 5+ tickets to bug tracker every day. It does NOT really help anyone or anything. Always ellaborate before posting anything. Moreover, your tickets (most of them) are just junk, like this one.

RSS Guard is almost one-man project developed in my free time and I do not want to sort these crappy bug reports for things which are clearly no bugs.

PLEASE.

For more verbose chit-chat, use our discord channel. Link is in readme, man.

**Comment by inson1** [comment]
@martinrotter sorry I didnt try to make junk, I will try elaborate more. 
But this isnt junk. I dont like this particular design choice, its so bad (sorry) that I thought it could be bug. 
I only posted 5 issues day, because I first tried the app. Yes, I created few junk ones ok, but your referenced issues arent junk. Not important yes, but not junk. Idk why use Discord, anybody can text here or there the same + if someone ask here even dumb question, then in 70+% nobody is going to ask it again.
